_team/2020-01-24-Vivek-Kumar.md

```diff
@@ -0,0 +1,9 @@
+---
+status: current
+photo: /assets/uploads/team/vivek_kumar.jpg
+full_name: Vivek Kumar
+position: Undergraduate Student Researcher
+email: vk2425@columbia.edu
+github: https://github.com/vivvyk
+---
+Vivek is an undergraduate senior studying machine learning at Columbia University. His main coursework and internship interests lie in computer vision, robotics, and deep learning.  At the lab, Vivek is working on digital detection of irrigation systems in Africa.
```

Comment:
You could change this to say irrigated lands or plots instead of irrgation systems. Technically this is correct by association.